app/controllers/campaigns/new_controller.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+class Campaigns::NewController < ApplicationController
```

Comment:
We usually call the other ones `*EditController` rather than `NewController` but this is okay too

Reply:
Yeah I did notice that, I decided to go with new because in the prototype there's a separate edit flow which I thought could be implemented with an edit controller in the future. Do you think it's worth renaming this here?

Reply:
Nope, that makes sense!